### PR TITLE
Analytics with AHOY

### PR DESCRIPTION
**Pull request by oliverochman**

### CODE REVIEW

File: src/components/Inquiry/InquiryCompanySize.jsx [hunk 1]
```diff
@@ -7,13 +7,14 @@ const InquiryCompanySize = () => {
...
   return (
-    <div data-cy='company-size-container'>
+    <div  data-cy='company-size-container'>
       <Question text={t('question.size')} />
       <Answer
         text={t('answer.size')}
         type='number'
         placeholder='5'
         questionKey='size'
+        data-section="company-size"
```
Comment (1CIM) [inline src/components/Inquiry/InquiryCompanySize.jsx]: formating?

Comment (tochman) [reply src/components/Inquiry/InquiryCompanySize.jsx]: Removed in latest commit

**Review by lukeperrin10 — APPROVED** [review]

File: src/components/Inquiry/InquiryCompanySize.jsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ const InquiryCompanySize = () => {
   const { t } = useTranslation()
 
   return (
-    <div data-cy='company-size-container'>
+    <div  data-cy='company-size-container'>
```
Comment (1CIM) [inline src/components/Inquiry/InquiryCompanySize.jsx]: nitpicking but this file wouldn’t be here unless that dubble space right?

**Review by 1CIM — APPROVED** [review]

File: src/components/Inquiry/InquiryCompanySize.jsx (hunk 2 above)
Comment (oliverochman) [reply src/components/Inquiry/InquiryCompanySize.jsx]: addressed

**Pull request merged**